src/SCRIPTS/BF/PAGES/rescue.lua

```diff
@@ -16,7 +16,12 @@ if apiVersion >= 1.41 then
     fields[#fields + 1] = { t = "Angle",            x = x,          y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 200, vals = { 1, 2 } }
     fields[#fields + 1] = { t = "Initial Altitude", x = x,          y = inc.y(lineSpacing), sp = x + sp, min = 20, max = 100, vals = { 3, 4 } }
     fields[#fields + 1] = { t = "Descent Distance", x = x,          y = inc.y(lineSpacing), sp = x + sp, min = 30, max = 500, vals = { 5, 6 } }
-    fields[#fields + 1] = { t = "Ground Speed",     x = x,          y = inc.y(lineSpacing), sp = x + sp, min = 30, max = 3000, vals = { 7, 8 } }
+
+    if apiVersion >= 1.45 then
+        fields[#fields + 1] = { t = "Ground Speed", x = x,          y = inc.y(lineSpacing), sp = x + sp, min = 30, max = 3000, vals = { 7, 8 } }
+    else
+        fields[#fields + 1] = { t = "Ground Speed", x = x,          y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 3000, vals = { 7, 8 }, scale = 100 }
+    end
```

Comment:
Did the min limit change? Hasn't it always been 0.3m/s? Are you sure you're scaling it for the right api version here?

Reply:
Min has changed. See below.

Reply:
It's still 0.3m/s in the configurator, but sure, we can do 0. Just need to decide if it's supposed to be in m/s for api version >= 1.45 or below

Reply:
@haslinghuis This has to be changed around. This adds the scaling for apiVersion < 1.45

Reply:
Fixed :)